cadasta/gui/tools/cadasta_login.py

```diff
@@ -60,15 +60,20 @@ def enable_button(self, custom_button):
         :type custom_button: QWidget
         """
         custom_button.setEnabled(True)
-        custom_button.setStyleSheet("background-color:#525252; cursor:pointer;" + CadastaStyle.button_style())
+        custom_button.setStyleSheet(
+                "background-color:#525252; cursor:pointer;" +
```

Comment:
We use single quote for strings : see https://github.com/inasafe/inasafe/wiki/Coding-Standards#strings-and-internationalisation